src/chi/components/tables/tables.scss

```diff
@@ -134,6 +134,60 @@ th {
     }
   }
 
+  &.-portal {
+    thead {
+      tr {
+        th {
+          background-color: set-color(grey, 25);
+          border-bottom: 0;
+          font-weight: 700;
+          padding-bottom: 0.75rem;
+          padding-top: 0.75rem;
```

Comment:
@janibolkvadze please remove this properties as the height of the th should be achieved with our helper sizes classes for tables. Same as we have done in the td.